xmipp

```diff
@@ -1220,22 +1246,20 @@ if __name__ == '__main__':
     elif mode=="all":
         Nproc = 8
         branch = ''
-        doPull = False
         buildDir = 'build'
         for arg in sys.argv[2:]:
             if arg.startswith("N="):
                 Nproc = int(arg[2:])
             elif arg.startswith("br="):
                 branch = arg[3:]
-                doPull = True
             elif arg.startswith("dir="):
                 buildDir = arg[4:]
             else:
                 usage("Unknown %s argument"%arg)
                 sys.exit(1)
 
         ok = (getDependencies()
-            and getSources(branch, doPull)
+            and getSources(branch)
             and config()
             and checkConfig()
```

Comment:
We could put the getDependencies() after checkConfig and, if CUDA=False, cuFFTAdvisor should not be download.

I know that this is not pretended in this PR, but just to take it in advanced.

Reply:
In addition, check CUDA variable in order to compile or not cuFFTAdvisor instead of check 'nvcc' in the PATH (in order to be able to skip cuda installation even with nvcc in the PATH).

Also, it's not related with this PR but it could be include.

Reply:
Good point. 
I agree, lets do it in another PR.

Reply:
May I consider this PR approved?